web/home.go

```diff
@@ -1,11 +1,37 @@
 package web
 
 import (
+	"fmt"
+	"html/template"
 	"net/http"
-
-	"github.com/gin-gonic/gin"
 )
 
-func homeHandler(c *gin.Context) {
-	c.HTML(http.StatusOK, "home.html.tmpl", gin.H{})
+// Index data is used for the home template
+type Index struct {
+	Title     string
+	Copyright string
+}
+
+func IndexHandler(templates map[string]template.Template) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		data := Index{
+			Title:     "SUSE Console for SAP Applications",
+			Copyright: "2019-2021 SUSE, all rights reserved.",
+		}
+
+		tmpl, ok := templates["home.html.tmpl"]
+
+		if !ok {
+			http.Error(w, fmt.Sprintf("The template HOME does not exist"),
+				http.StatusInternalServerError)
+			return
+		}
+
+		err := tmpl.ExecuteTemplate(w, "base", data)
+		if err != nil {
+			http.Error(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+
```

Comment:
I think we're missing to set the http status code and the `content-type` header, which was something that the `gin` render took care for us. How do we do this with `chi`?